transit/models/order_details.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+from transit.models.base import BaseModel
+from transit.models.customer import Customer
+from transit.models.item import (
+    Item,
+    ItemDetails
+)
+
+
+class OrderDetails(BaseModel):
+    order_details_id = models.CharField(
+        max_length=50, primary_key=True, unique=True, db_column='OrderDetailsID'
+    )
+
+    customer = models.ForeignKey(Customer, models.DO_NOTHING, db_column='CustomerID')
+    order_received_date = models.CharField(
+        db_column='OrderReceivedDate', max_length=255, blank=True, null=True
+    )
+
+    class Meta:
+        managed = True
+        db_table = 'OrderDetails'
+
+
+class OrderLineDetails(BaseModel):
+    order_details = models.ForeignKey(OrderDetails, models.DO_NOTHING, db_column='OrderDetailsID')
+
+    # TODO - product (item) and item details could be fixed, we could keep one of them only
+    product = models.ForeignKey(Item, models.DO_NOTHING, db_column='ProductId')
```

Comment:
I don't see ProductId column in Item model, is this intentional?

Reply:
Yes. In the legacy files in 'T_orderLineDetails' you can find 'ProductMasterId' and this relates to 'Item'. Unfortunately we have 'inconsistency' in the 'namings' of fields.

Reply:
In 'Item' (legacy file) the PK is called 'ProductMasterID'

Reply:
Thanks for explanation